### PR TITLE
Feature: Implement Firebase Cloud Functions httpsCallable() Support

### PR DESCRIPTION
**Pull request by grandnexus**
Hi everyone, I have implemented firebase cloud functions httpscallable with test code and example.

### CODE REVIEW

**Comment by kevmoo** [comment]
Excited to see this!

Please revert the whitespace changes to the HTML files. It makes reviewing much harder.

**Comment by grandnexus** [comment]
> Excited to see this!
> 
> Please revert the whitespace changes to the HTML files. It makes reviewing much harder.

Noted. I have reverted whitespaces in HTML files.

File: example/index.html [hunk 1]
```diff
@@ -4,19 +4,20 @@
   <meta charset="UTF-8">
   <title>Examples</title>
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
-        integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```
Comment (kevmoo) [inline example/index.html]: here too

File: example/messaging/index.html [hunk 2]
```diff
@@ -1,59 +1,58 @@
...
+  <div class="container">
     <h1>Messaging demo</h1>
     <ul>
-        <li>
-            <!-- This is autofill the token come from the firebase sdk -->
-            <label>
-                Token
-                <input disabled id="token" />
-            </label>
-        </li>
```
Comment (kevmoo) [inline example/messaging/index.html]: ditto

File: test/app_test.html [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
-    <script src="https://www.gstatic.com/firebasejs/5.9.2/firebase-storage.js"></script>
-    <script src="packages/test/dart.js"></script>
-    <link rel="x-dart-test" href="app_test.dart">
-    </body>
-</html>
+<head>
+  <title>App Test</title>
+</head>
+<body>
+  <script src="https://www.gstatic.com/firebasejs/5.9.3/firebase-app.js"></script>
```
Comment (kevmoo) [inline test/app_test.html]: ditto

File: example/index.html (hunk 1 above)
Comment (grandnexus) [reply example/index.html]: Done

File: example/messaging/index.html (hunk 2 above)
Comment (grandnexus) [reply example/messaging/index.html]: Done

File: test/app_test.html (hunk 3 above)
Comment (grandnexus) [reply test/app_test.html]: Done

**Comment by grandnexus** [comment]
> still a lot of whitespace to clean up

Done.

**Comment by kevmoo** [comment]
@grandnexus – I'm going to close this PR for now.

Please open another PR once you have just the changes you want in. Ideally very few files will be changed, etc.

Thanks!

**Pull request closed without merging**